_Boards/RaspberryPi/Pico/raspberry_pico.board.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
We haven't checked the .board.json file into this github repo for any of the amtel platforms so this should probably be excluded from this PR.

Reply:
Ups, you are rigth. Was the best location for me during delelpment.
I will delete it.